OvmfPkg/PlatformPei/PlatformId.c

```diff
@@ -0,0 +1,660 @@
+/**@file
+  PlatformId Event HOB creation
+
+  Copyright (c) 2024, Google LLC. All rights reserved.<BR>
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Base.h>
+#include <Guid/TcgEventHob.h>
+#include <IndustryStandard/UefiTcgPlatform.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/BaseLib.h>
+#include <Library/DebugLib.h>
+#include <Library/HobLib.h>
+#include <Library/PeiServicesLib.h>
+#include <Library/QemuFwCfgLib.h>
+
+#define HEX_DIGITS  "0123456789abcdef"
+#define DPREFIX     "sp800155evts: "
+
+/**
+ * TCG specifies a locator type with the following values
+ * 0 - Raw data in the locator itself.
+ * 1 - URI in rtf2396 format.
+ * 2 - local device path in EFI_DEVICE_PATH_PROTOCOL format.
+ * 3 - UEFI variable (EFI_GUID, 00-terminated UCS2 string)
+ *
+ * Each of these could potentially be validated before registering, but they
+ * aren't necessarily loaded in the PEI phase this runs in.
+**/
+#define LOCATOR_TYPE_RAW_DATA       0
+#define LOCATOR_TYPE_URI            1
+#define LOCATOR_TYPE_DEVICE_PATH    2
+#define LOCATOR_TYPE_UEFI_VARIABLE  3

```

Comment:
I think it is defined by TCG PFP, right? If so, please move the definition to UefiTcgPlatform.h.

I also recommend to use `TCG_` as name space prefix.

Reply:
Done.